602015334451,(psycopg2.ProgrammingError) subquery must return only one column
LINE 1: ...018-09-25T18:25:21.821303+00:00'::timestamptz, 0, (SELECT se...
                                                             ^
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, (SELECT seller.id AS seller_id, seller.created_at AS seller_created_at, seller.created_by AS seller_created_by, seller.updated_at AS seller_updated_at, seller.updated_by AS seller_updated_by, seller.code AS seller_code, seller."time_duing_DS" AS "seller_time_duing_DS", seller.segment_id AS seller_segment_id, seller.user_id AS seller_user_id, seller.goal AS seller_goal, seller.commission AS seller_commission \nFROM seller \nWHERE seller.code = %(code_1)s), %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'tax': '287.39', 'client_id': None, 'order_number': 1208, 'created_by': 1, 'date': '2018-09-25T13:42:10-05:00', 'total': '1800.00', 'updated_by': 0, 'updated_at': datetime.datetime(2018, 9, 25, 18, 25, 21, 821303, tzinfo=<UTC>), 'status': 'ordered', 'shipping': '0.00', 'created_at': datetime.datetime(2018, 9, 25, 18, 25, 21, 821290, tzinfo=<UTC>), 'code_1': 'heidy-523'}] (Background on this error at: http://sqlalche.me/e/f405)
601936986163,(psycopg2.ProgrammingError) subquery must return only one column
LINE 1: ...018-09-25T18:25:21.789361+00:00'::timestamptz, 0, (SELECT se...
                                                             ^
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, (SELECT seller.id AS seller_id, seller.created_at AS seller_created_at, seller.created_by AS seller_created_by, seller.updated_at AS seller_updated_at, seller.updated_by AS seller_updated_by, seller.code AS seller_code, seller."time_duing_DS" AS "seller_time_duing_DS", seller.segment_id AS seller_segment_id, seller.user_id AS seller_user_id, seller.goal AS seller_goal, seller.commission AS seller_commission \nFROM seller \nWHERE seller.code = %(code_1)s), %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'tax': '1197.48', 'client_id': None, 'order_number': 1207, 'status': 'ordered', 'date': '2018-09-25T11:53:42-05:00', 'total': '7500.00', 'updated_by': 0, 'updated_at': datetime.datetime(2018, 9, 25, 18, 25, 21, 789361, tzinfo=<UTC>), 'created_at': datetime.datetime(2018, 9, 25, 18, 25, 21, 789346, tzinfo=<UTC>), 'shipping': '0.00', 'created_by': 1, 'code_1': 'xiomara-8yr '}] (Background on this error at: http://sqlalche.me/e/f405)
601936986163,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (25, 2018-09-25 12:27:11.79137-05, 1, 2018-09-25 12:27:11.791388-05, 0, null, null, 2018-09-25 11:53:42-05, 1207, 7500.00, 0.00, 1197.48, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'created_by': 1, 'client_id': None, 'shipping': '0.00', 'date': '2018-09-25T11:53:42-05:00', 'created_at': datetime.datetime(2018, 9, 25, 17, 27, 11, 791370, tzinfo=<UTC>), 'tax': '1197.48', 'status': 'ordered', 'seller_id': None, 'order_number': 1207, 'total': '7500.00', 'updated_at': datetime.datetime(2018, 9, 25, 17, 27, 11, 791388, tzinfo=<UTC>), 'updated_by': 0}] (Background on this error at: http://sqlalche.me/e/gkpj)
601936986163,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (24, 2018-09-25 12:25:31.871954-05, 1, 2018-09-25 12:25:31.872018-05, 0, null, null, 2018-09-25 11:53:42-05, 1207, 7500.00, 0.00, 1197.48, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'created_by': 1, 'seller_id': None, 'order_number': 1207, 'created_at': datetime.datetime(2018, 9, 25, 17, 25, 31, 871954, tzinfo=<UTC>), 'tax': '1197.48', 'updated_by': 0, 'client_id': None, 'date': '2018-09-25T11:53:42-05:00', 'shipping': '0.00', 'updated_at': datetime.datetime(2018, 9, 25, 17, 25, 31, 872018, tzinfo=<UTC>), 'total': '7500.00', 'status': 'ordered'}] (Background on this error at: http://sqlalche.me/e/gkpj)
597205090355,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (13, 2018-09-19 10:36:55.501169-05, 1, 2018-09-19 10:36:55.501185-05, 0, null, null, 2018-09-19 08:19:51-05, 1180, 59900.00, 0.00, 9563.87, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1180, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '59900.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 501185, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 501169, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-19T08:19:51-05:00', 'tax': '9563.87', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
597202337843,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (12, 2018-09-19 10:36:55.46335-05, 1, 2018-09-19 10:36:55.463368-05, 0, null, null, 2018-09-19 08:14:04-05, 1179, 15900.00, 0.00, 2538.66, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1179, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '15900.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 463368, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 463350, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-19T08:14:04-05:00', 'tax': '2538.66', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596977057843,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (11, 2018-09-19 10:36:55.441037-05, 1, 2018-09-19 10:36:55.441064-05, 0, null, null, 2018-09-18 22:49:02-05, 1178, 15900.00, 0.00, 2538.66, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1178, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '15900.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 441064, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 441037, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T22:49:02-05:00', 'tax': '2538.66', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596862861363,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (9, 2018-09-19 10:36:55.347106-05, 1, 2018-09-19 10:36:55.347123-05, 0, null, null, 2018-09-18 19:38:24-05, 1176, 7800.00, 0.00, 1245.38, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1176, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '7800.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 347123, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 347106, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T19:38:24-05:00', 'tax': '1245.38', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596826685491,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (8, 2018-09-19 10:36:55.251022-05, 1, 2018-09-19 10:36:55.251038-05, 0, null, null, 2018-09-18 18:41:44-05, 1174, 23900.00, 0.00, 3815.97, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1174, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '23900.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 251038, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 251022, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T18:41:44-05:00', 'tax': '3815.97', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596813873203,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (7, 2018-09-19 10:36:55.143775-05, 1, 2018-09-19 10:36:55.14379-05, 0, null, null, 2018-09-18 18:19:44-05, 1171, 0.00, 0.00, 0.00, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1171, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '0.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 143790, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 143775, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T18:19:44-05:00', 'tax': '0.00', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596811448371,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (6, 2018-09-19 10:36:55.110088-05, 1, 2018-09-19 10:36:55.110102-05, 0, null, null, 2018-09-18 18:15:39-05, 1170, 7500.00, 0.00, 1197.48, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1170, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '7500.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 110102, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 110088, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T18:15:39-05:00', 'tax': '1197.48', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596753514547,(psycopg2.IntegrityError) null value in column "seller_id" violates not-null constraint
DETAIL:  Failing row contains (5, 2018-09-19 10:36:55.081677-05, 1, 2018-09-19 10:36:55.081696-05, 0, null, null, 2018-09-18 16:39:20-05, 1168, 99500.00, 0.00, 15886.55, ordered).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'order_number': 1168, 'updated_by': 0, 'client_id': None, 'shipping': '0.00', 'total': '99500.00', 'updated_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 81696, tzinfo=<UTC>), 'created_by': 1, 'created_at': datetime.datetime(2018, 9, 19, 15, 36, 55, 81677, tzinfo=<UTC>), 'status': 'ordered', 'date': '2018-09-18T16:39:20-05:00', 'tax': '15886.55', 'seller_id': None}] (Background on this error at: http://sqlalche.me/e/gkpj)
596649639987,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (167, 2018-09-18 16:22:21.128437-05, 1, 2018-09-18 16:22:21.12846-05, 0, 5, null, 2018-09-18 14:16:44-05, ordered, 1156, 8900.00, 1421.01, 0.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '0.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 128460, tzinfo=<UTC>), 'order_number': 1156, 'status': 'ordered', 'tax': '1421.01', 'client_id': None, 'created_by': 1, 'date': '2018-09-18T14:16:44-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 128437, tzinfo=<UTC>), 'total': '8900.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
596405518387,The phone number into order is not a valid number
596171915315,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (166, 2018-09-18 16:22:21.060827-05, 1, 2018-09-18 16:22:21.060872-05, 0, 5, null, 2018-09-18 10:39:52-05, ordered, 1149, 0.00, 0.00, 0.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '0.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 60872, tzinfo=<UTC>), 'order_number': 1149, 'status': 'ordered', 'tax': '0.00', 'client_id': None, 'created_by': 1, 'date': '2018-09-18T10:39:52-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 60827, tzinfo=<UTC>), 'total': '0.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
596168769587,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (165, 2018-09-18 16:22:21.002097-05, 1, 2018-09-18 16:22:21.002127-05, 0, 5, null, 2018-09-17 23:30:50-05, ordered, 1148, 8900.00, 1421.01, 0.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '0.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 2127, tzinfo=<UTC>), 'order_number': 1148, 'status': 'ordered', 'tax': '1421.01', 'client_id': None, 'created_by': 1, 'date': '2018-09-17T23:30:50-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 21, 2097, tzinfo=<UTC>), 'total': '8900.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
596155629619,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (164, 2018-09-18 16:22:20.944554-05, 1, 2018-09-18 16:22:20.944592-05, 0, 5, null, 2018-09-18 10:40:09-05, ordered, 1147, 0.00, 0.00, 0.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '0.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 944592, tzinfo=<UTC>), 'order_number': 1147, 'status': 'ordered', 'tax': '0.00', 'client_id': None, 'created_by': 1, 'date': '2018-09-18T10:40:09-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 944554, tzinfo=<UTC>), 'total': '0.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
596047036467,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (163, 2018-09-18 16:22:20.896612-05, 1, 2018-09-18 16:22:20.896636-05, 0, 5, null, 2018-09-17 20:17:38-05, ordered, 1145, 2800.00, 447.06, 0.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '0.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 896636, tzinfo=<UTC>), 'order_number': 1145, 'status': 'ordered', 'tax': '447.06', 'client_id': None, 'created_by': 1, 'date': '2018-09-17T20:17:38-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 896612, tzinfo=<UTC>), 'total': '2800.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
596036649011,(psycopg2.IntegrityError) null value in column "client_id" violates not-null constraint
DETAIL:  Failing row contains (162, 2018-09-18 16:22:20.840224-05, 1, 2018-09-18 16:22:20.84027-05, 0, 5, null, 2018-09-17 20:00:28-05, ordered, 1144, 22900.00, 3656.30, 3000.00).
 [SQL: 'INSERT INTO "order" (created_at, created_by, updated_at, updated_by, seller_id, client_id, date, status, order_number, total, tax, shipping) VALUES (%(created_at)s, %(created_by)s, %(updated_at)s, %(updated_by)s, %(seller_id)s, %(client_id)s, %(date)s, %(status)s, %(order_number)s, %(total)s, %(tax)s, %(shipping)s) RETURNING "order".id'] [parameters: {'updated_by': 0, 'shipping': '3000.00', 'seller_id': 5, 'updated_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 840270, tzinfo=<UTC>), 'order_number': 1144, 'status': 'ordered', 'tax': '3656.30', 'client_id': None, 'created_by': 1, 'date': '2018-09-17T20:00:28-05:00', 'created_at': datetime.datetime(2018, 9, 18, 21, 22, 20, 840224, tzinfo=<UTC>), 'total': '22900.00'}] (Background on this error at: http://sqlalche.me/e/gkpj)
593491296307,The phone number into order is not a valid number
593464393779,The phone number into order is not a valid number
